config/packages/doctrine.yaml

```
@@ -22,6 +22,11 @@ doctrine:
                 dir: '%kernel.project_dir%/app/Entity'
                 prefix: 'App\Entity'
                 alias: App
+        dql:
+            string_functions:
+                DAY: 'App\Doctrine\Day'
+                MONTH: 'App\Doctrine\Month'
+                YEAR: 'App\Doctrine\Year'
```

Let's use either full official namespace https://github.com/doctrine/orm/tree/3.3.x/src/Query/AST/Functions or what sylius has https://github.com/Sylius/Sylius/blob/2.1/src/Sylius/Bundle/CoreBundle/Doctrine/DQL/Month.php

Or maybe `Doctrine\Query\Function` as alternative


```suggestion
                DAY: 'App\Doctrine\Query\AST\Function\Day'
                MONTH: 'App\Doctrine\Query\AST\Function\Month'
                YEAR: 'App\Doctrine\Query\AST\Function\Year'
```

---

done :)